qutip/random_objects.py

```diff
@@ -319,13 +319,22 @@ def rand_ket(N=0, density=1, dims=None, seed=None):
     oper : qobj
         Nx1 ket state quantum operator.
 
+    Raises
+    -------
+    ValueError
+        Specify either the number of rows of operator (N) or
+        dimensions of quantum object (dims).
```

Comment:
For the "raises" section of the documentation, the description should be the condition that the exception is raised, rather than a description of how to fix it (see [the numpydoc documentation](https://numpydoc.readthedocs.io/en/latest/format.html#raises)).  For example,
```
Raises
------
ValueError
    If neither ``N`` or ``dims`` are specified.
```